auslib/web/public/api.yml

```diff
@@ -587,3 +587,210 @@ paths:
       responses:
         200:
           description: version.json content.
+
+# Public Rules endpoints
+  /rules:
+    get:
+      operationId: "auslib.web.public.rules.get"
+      description: ""
+      produces: ["application/json"]
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/RulesWrapper"
+
+  /rules/{id_or_alias}:
+    get:
+      operationId: "auslib.web.public.rules.get_by_id_or_alias"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: id_or_alias
+          in: path
+          required: true
+          type: string
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/Rule"
+
+  /rules/{rule_id}/revisions:
+    get:
+      operationId: "auslib.web.public.rules.get_revisions"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: rule_id
+          in: path
+          required: true
+          type: integer
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/RuleHistoryWrapper"
+
+  /releases:
+    get:
+      operationId: "auslib.web.public.releases.get"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: product
+          in: query
+          required: false
+          type: string
+        - name: name_prefix
+          in: query
+          required: false
+          type: string
+        - name: names_only
+          in: query
+          required: false
+          type: string
+      responses:
+        200:
+          description: ""
+          schema:
+            $ref: "#/definitions/RulesWrapper"
+
+  /releases/{release}:
+    get:
+      operationId: "auslib.web.public.releases.get_release"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: release
+          in: path
+          required: true
+          type: string
+        - name: pretty
+          in: query
+          required: false
+          type: integer
+      responses:
+        200:
+          description: "Release Blob"
+
+  /releases/{release}/revisions:
+    get:
+      operationId: "auslib.web.public.releases.get_revisions"
+      description: ""
+      produces: ["application/json"]
+      parameters:
+        - name: release
+          in: path
+          required: true
+          type: string
+      responses:
+        200:
+          description: "Release History"
+
+definitions:
```

Comment:
Hmm, it looks like you can pass a dict as the "specification" (https://github.com/zalando/connexion/blob/93c06711ed02b453121818f65b25ac3c863aefdd/connexion/apps/abstract.py#L97). I wonder if we can put the definitions in a common file, and manually combine that + the endpoints instead of passing a filename?